speaksfer/urls.py

```diff
@@ -15,7 +15,31 @@
 """
 from django.contrib import admin
 from django.urls import path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Speaksfer API",
+        default_version="v1",
+        description="Test description",
```

Comment:
Also hii usiite test give it a description

Reply:
Sawa